tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -250,8 +255,27 @@ func (rb *PSATestSuite) ValidateAdditionalPSA(role string) {
 	}
 }
 
+func (rb *PSATestSuite) ValidateEditCluster(role string) {
+	log.Infof("Editing the cluster as %v", role)
+	clusterType, err := editCluster(rb.standardUserClient, rb.clusterName, rb.namespace, "")
+	switch role {
+	case roleOwner, restrictedAdmin:
+		require.NoError(rb.T(), err)
```

Comment:
Can we add a check to ensure psact is disabled on the cluster? probably creating a deployment and making sure its active?

Reply:
I added a check in the rbac.go to ensure that psact value is indeed changed after editing the cluster